### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by soniasingla**

### CODE REVIEW

File: pkg/controllers/cstorvolumeconfig/controller_test.go [hunk 1]
```diff
@@ -71,7 +71,6 @@ type fixture struct {
 
 	// Actions expected to happen on the client. Objects from here are also
 	// preloaded into NewSimpleFake.
-	actions        []core.Action
```
Comment (mittachaitu) [inline pkg/controllers/cstorvolumeconfig/controller_test.go:74]: Is linter shown this field as unused?

Comment (soniasingla) [reply pkg/controllers/cstorvolumeconfig/controller_test.go:74]: yes!

Comment (mittachaitu) [reply pkg/controllers/cstorvolumeconfig/controller_test.go:74]: But we are using them as we can see [here](https://github.com/openebs/cstor-operators/pull/301/checks?check_run_id=2500796432) test cases are failing.

Comment (mittachaitu) [reply pkg/controllers/cstorvolumeconfig/controller_test.go:74]: We can undo the changes done to `actions`

**Comment by mittachaitu** [comment]
@soniasingla Thanks for the PR but changes in this PR are already taken into master via #290 so closing the PR.

**Pull request closed without merging**